server/src/main/java/io/druid/client/CachingClusteredClient.java

```diff
@@ -112,6 +112,10 @@ public CachingClusteredClient(
     this.cacheConfig = cacheConfig;
     this.backgroundExecutorService = MoreExecutors.listeningDecorator(backgroundExecutorService);
 
+    if (cacheConfig.isQueryCacheable(Query.GROUP_BY)) {
+      log.warn("Caching for groupBys is enabled, but they won't be cached if they are executed with the strategy v2!");
```

Comment:
can you also clarify that they won't be cached on druid brokers also an advise to enable caching on data nodes for V2 strategy would be helpful.